gapic/src/main/com/google/gapid/perfetto/models/Tracks.java

```diff
@@ -50,12 +51,15 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.logging.Logger;
 import java.util.stream.Collectors;
 
 /**
  * Determines what tracks to show for a trace.
  */
 public class Tracks {
+  private static final Logger LOG = Logger.getLogger(CounterInfo.class.getName());
```

Comment:
By convention this logger is shared by all things within this class, so I think maybe the input parameter should be `Tracks.class.getName()` if you are really using the logger.

Reply:
Nice catch. Updated it now.